app_dart/test/request_handlers/refresh_cirrus_status_test.dart

```diff
@@ -104,20 +150,47 @@ void main() {
       config.db.values[task.key] = task;
 
       expect(task.status, 'New');
+      branchHttpClient.request.response.body = branchRegExp;
       await tester.get(handler);
       expect(task.status, 'In Progress');
     });
+
+    test('skip updating cirrus status when there is no matching branch',
```

Comment:
What about adding a test for a matching branch different than master?

Reply:
Added.